<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HospitalizaciaEdit</class>
 <widget class="QWizardPage" name="HospitalizaciaEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editácia hospitalizácie</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nemocnicaLabel">
     <property name="text">
      <string>Nemocnica</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="nemocnicaDisplay">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="odLabel">
     <property name="text">
      <string>Hospitalizovaný od</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="odEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="doLabel">
     <property name="text">
      <string>Hospitalizovaný do</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateEdit" name="doEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ukoncenieCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="diagnozaLabel">
     <property name="text">
      <string>&amp;Diagnóza</string>
     </property>
     <property name="buddy">
      <cstring>diagnozaEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPlainTextEdit" name="diagnozaEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ukoncenieCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>doEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
